package.json

```diff
@@ -121,5 +121,8 @@
     "*.{json,yaml,yml,md}": [
       "prettier --write"
     ]
+  },
+  "publishConfig": {
+    "tag": "alpha"
```

Comment:
please remove it until latest version setting will be added to action
this way every new version will publish like alpha